rpcserver/rpc_server.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
Prefer to change the `rpcserveer` subpackage to `services/service` subpackage. What do you think? @djshow832 @bb7133 @zimulala

Reply:
How about `services/rpc_server`?

Reply:
Already move to `services/rpcserver`

Reply:
Already move to `server` pkg, no need a new pkg, Thanks

Reply:
And change the PR title.